documentation/concepts/00_accounts.md

```diff
@@ -104,13 +104,17 @@ Given global instantiated Aleo parameters and subroutines.
 1. Sample a 32 byte `seed` from random
     
 2. Construct private key components
-    - `sk_sig` = Blake2s(`seed`, 0)
-    - `sk_prf` = Blake2s(`seed`, 1)
-    - `r_pk` = Blake2s(`seed`, `counter`)
+    - `sk_sig` = BLAKE2s(`seed` | 0), where | denotes concatenation
+    - `sk_prf` = BLAKE2s(`seed` | 1)
+    - `r_pk` = BLAKE2s(`seed` | `counter`)
            
```

Comment:
I would move 'where `|` denotes concatenation right here, in its own line, after the itemized lists.

Reply:
I would also add 'and where `BLAKE2s` denotes unkeyed BLAKE2s-256, as defined in [RFC 7693](https://www.rfc-editor.org/rfc/rfc7693).